ufc/tests/test-case-integer-flag.json

```diff
@@ -194,7 +194,7 @@
     {
       "subjectKey": "2",
       "subjectAttributes": {},
-      "assignment": 2,
+      "assignment": 99,
```

Comment:
I assume this was done to see if the tests would fail and I see in the github actions log that they do.

Reply:
yes- reverted